### PR TITLE
Bug 2308101: logrotate: add logrotate functionality for csi operator

### PR DESCRIPTION
**Pull request by parth-gr**
currently client operator has started using csi operator so collecting the csi logs from a new host path
hat csi-operator supports

### CODE REVIEW

**Comment by parth-gr** [comment]
once this is merged I will manually update the 4.17 backport with this commit

**Comment by parth-gr** [comment]
can we get a /lgtm approval?

**Review by yati1998 — APPROVED** [review]

**Comment by black-dragon74** [comment]
@parth-gr please address the suggested changes.

**Comment by parth-gr** [comment]
> @parth-gr please address the suggested changes.

sorry are you looking fro something else?

**Pull request merged**
